forc-plugins/forc-client/src/ops/run/cmd.rs

```diff
@@ -1,11 +1,10 @@
-use crate::ops::forc_run;
-use anyhow::{bail, Result};
 use clap::Parser;
 
 /// Run script project.
 /// Crafts a script transaction then sends it to a running node.
 #[derive(Debug, Default, Parser)]
-pub struct Command {
+#[clap(bin_name = "forc run")]
```

Comment:
Perhaps we want `version` here too like we have for the deploy command?
```suggestion
#[clap(version)]
#[clap(bin_name = "forc run")]
```
Possibly can also be written as:
```rust
#[clap(bin_name = "forc run", version)]
```
though I'm not sure